### ISSUE TITLE
how to disable a error in a line?

### ISSUE THREAD

**Issue by budarin** [SonarSource/SonarJS#1257]
How to disable error for the line of code?
There is no any info about disabling error in code

**Example**
```
passwordError: 'Password does not meet required complexity',
```
it raises the error `Review this potentially hardcoded credential.`

**Comment by vilchik-elena** [comment]
@budarin mark issue as false positive in SonarQube UI.

**Comment by vilchik-elena** [comment]
Try marking it as "won't fix" (see [doc](https://docs.sonarqube.org/latest/user-guide/security-reports/))

**Comment by budarin** [comment]
@vilchik-elena 
Elena, is it a magic comment? 
I dont understand what does it mean: "The Won't Fix designation ..."

**Comment by vilchik-elena** [comment]
@budarin are you using SonarQube? in SonarQube UI when reviewing issue, you can resolve it with 'won't fix' (e.g. see [issue](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.javascript%3Ajavascript&issues=AWik3aF576m-uU-YJ5UV&open=AWik3aF576m-uU-YJ5UV))
![image](https://user-images.githubusercontent.com/10108186/59348943-bceeaa00-8d18-11e9-8029-36e91f131222.png)

**Comment by budarin** [comment]
I use an extention SonarLint for VSCode14:49, 12 июня 2019 г., Elena Vilchik <notifications@github.com>:@budarin are you using SonarQube? in SonarQube UI when reviewing issue, you can resolve it with 'won't fix' (e.g. see issue)


—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.
С уважением Бударин Вадим

**Comment by vilchik-elena** [comment]
@budarin thank, then indeed you don't have SonarQube UI.
Try adding comment `// NOSONAR` at that line.

Anyway that's weird, I can't find such rule (and rule key would not hurt when reporting an issue, [template](https://github.com/SonarSource/SonarJS/issues/new) is for a reason here).

Are you sure you are using JS? It seams to me it's a TypeScript rule. Anyway please be richer in details next time.

**Comment by budarin** [comment]
adding comment // NOSONAR does not help - the error is still present

I do use typescript

**Comment by budarin** [comment]
@vilchik-elena 
I have moved to use it with tslint - it solves the problem
